Makefile

```diff
@@ -0,0 +1,31 @@
+MSBUILD = xbuild
+MDOC = mdoc/bin/Release/mdoc.exe
+
+all: build
+
+build: $(MDOC)
+$(MDOC): external/SharpZipLib/bin/Release/ICSharpCode.SharpZipLib.dll external/Mono.Cecil/bin/net_4_0_Release/Mono.Cecil.dll
+	$(MSBUILD) apidoctools.sln /p:Configuration=Release
+
+prepare:
+	git submodule update --init --recursive
+	nuget restore -PackagesDirectory Packages
+
+external/SharpZipLib/bin/Release/ICSharpCode.SharpZipLib.dll:
```

Comment:
In keeping with a top-level `bin` directory, this should be:

    bin/ICSharpCode.SharpZipLib.dll:
        -mkdir -p "`dirname "$@"`"
        $(MSBUILD) external/SharpZipLib/ICSharpCode.SharpZipLib.NET45/ICSharpCode.SharpZipLib.csproj /p:Configuration=Release
        cp external/SharpZipLib/bin/Release/ICSharpCode.SharpZipLib.dll "$@"